TRAF4.py

```diff
@@ -0,0 +1,176 @@
+from Bio import SeqIO
+from Bio.SeqUtils import gc_fraction
+
+
+class NuclAcidnucleotideError(ValueError):
+    """Custom error for NucleicAcidSequence classes"""
+    pass
+
+
+class BiologicalSequence(str):
```

Comment:
В этой дз такое вроде было разрешено, но в целом лучше не наследоваться от встроенных типов данных. Подробнее об этом можно посмотреть в консультации от 28 февраля (примерно 28-я минута)

```suggestion
class BiologicalSequence():
```